src/Symfony/Component/Form/Tests/Extension/Core/DataTransformer/DayOfWeekTransformerTest.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Tests\Extension\Core\DataTransformer;
+
+use Symfony\Component\Form\Extension\Core\DataTransformer\DayOfWeekTransformer;
+
+class DayOfWeekTransformerTest extends \PHPUnit_Framework_TestCase
+{
+    private $transformer;
+
+    private $oldLocale;
+
+    protected function setUp()
+    {
+        $this->transformer = new DayOfWeekTransformer();
+        $this->oldLocale = \Locale::getDefault();
+    }
+
+    protected function tearDown()
+    {
+        $this->transformer = null;
+        \Locale::setDefault($this->oldLocale);
+    }
+
+    /**
+     * @dataProvider getValidData
+     */
+    public function testTransforms($locale, $pattern, $in, $out)
+    {
+        \Locale::setDefault($locale);
+        $transformer = new DayOfWeekTransformer($pattern);
+
+        $this->assertEquals($out, $transformer->transform($in));
+    }
+
+    /**
+     * @dataProvider getValidData
+     */
+    public function testReverseTransforms($locale, $pattern, $in, $out)
+    {
+        \Locale::setDefault($locale);
+        $transformer = new DayOfWeekTransformer($pattern);
+
+        $this->assertEquals($in, $transformer->reverseTransform($out));
+    }
+
+    public function getValidData()
+    {
+        return array(
+            array('fr_FR', 'eeee', 7, 'dimanche'),
+            array('en_US', 'eeee', 7, 'Sunday'),
```

Comment:
Because it actually is like it is supposed to be. Same thing for months: January vs janvier. This tests that the difference works, but is not an exhaustive test of the Intl library.